perllib/FixMyStreet/Cobrand/Brent.pm

```diff
@@ -547,6 +542,40 @@ sub waste_munge_request_form_data {
     }
 }
 
+sub available_permissions {
+    my $self = shift;
+
+    my $perms = $self->next::method();
+    $perms->{Waste}->{can_pay_with_csc} = "Can use CSC to pay for subscriptions";
+
+    return $perms;
+}
+
+sub waste_staff_choose_payment_method { 1 }
```

Comment:
Do we want this function to be 1? It adds the cheque box, but then next function is returning 0. I think given there's only CC payment, we want this to be 0.